addons/info/README.md

```diff
@@ -62,6 +62,14 @@ storiesOf('Component', module)
   )
 ```
 
+## Usage as decorator
+
+It is possible to add infos by default to all components by using a global or story decorator. The drawback is you won't be able to display a distinct info message per story.
+
+```
+addDecorator(withInfo()(story => story()));
```

Comment:
Wouldn't the info addon show information on the `WrapStory` component instead of the actual story component with this?